src/averageAges.js

```diff
@@ -14,12 +14,20 @@
  *
  * @return {number}
  */
+function averageAge(age) {
```

Comment:
Function names should start with verb (action)
```suggestion
function getAverageAge(age) {
```